adoc/bp_postgresql_database_migration.adoc

```diff
@@ -4,103 +4,182 @@
 include::entities.adoc[]
 
 
-{productname} 3 uses postgresql database version 9.4.
-Postgresql version 9.6 has been officially released for {sls} 12 SP3.
-In the near future postgresql 9.6 will become the base version provided by {productname}.
-Currently version 9.4 is hardcoded into SUSE Manager, therefore when installing {productname} it will explicitly use this version.
-This chapter provides guidance on migrating an existing 9.4 database to 9.6 on your {productname} Server.
+{productname} 3 uses PostgreSQL version 9.4.
+{productname} 3.2 uses PostgreSQL version 10.
+
+PostgreSQL version 9.6 has been released for {sls} 12 SP3.
+PostgreSQL version 10 has been released for {sls} 12 SP4 and later.
+This chapter provides guidance on migrating your {productname} Server from PostgreSQL 9.4 to 9.6, or migrating from 9.6 to 10.
 
 
 [[bp.postgresql.database.migration.new.installations]]
 == New {productname} Installations
 
 
-Once support for postgresql version 9.6 has been officially released for SUSE Manager, no action will be required for new installations.
-The {productname} extension will pick up the latest version during installation on SLES 12 SP3.
+PostgreSQL version 10 is now supported on {productname}.
+New {productname} installations will have PostgreSQL version 10 installed by default.
+When you install {productname} on {sle}{nbsp}12 SP4, the latest version of PostgreSQL will be installed by default.
 This will be fully transparent to the user.
-Check the active postgresql version with the following command:
+Check the active PostgreSQL version with this command:
 
 ----
 suse-manager-example-srv:~ # psql --version
-psql (PostgreSQL) 9.6.3
+psql (PostgreSQL) 10.9
 ----
 
+
+
 [[bp.postgresql.database.migrating.existing.installations]]
 == Migrating an Existing Installation
 
+[WARNING]
+====
+Always create a database backup before performing a migration.
+For more information about a database backup, see xref:bp_chap_suma_backup.adoc[].
+====
+
+PostgreSQL upgrades can be performed in two ways: a regular upgrade, or a fast upgrade:
+
+A regular upgrade will create a complete copy of the database, so you will need double the existing database size of space available.
+Regular upgrades can take a considerable amount of time, depending on the size of the database and the speed of the storage system.
+Regular upgrades also require database downtime, your database will not be accessible while the upgrade is running.
```

Comment:
I would delete this sentence as it could provoke false expectations! Also with a fast upgrade, the database needs to be down! Just not that long :)